### PR TITLE
generate enclave signer's key on the fly, and remove hardcoded ones

### PR DESCRIPTION
**Pull request by bvavala**
Signed-off-by: Bruno Vavala <bruno.vavala@intel.com>

### CODE REVIEW

File: ecc_enclave/enclave/CMakeLists-common-app-enclave.txt [hunk 1]
```diff
@@ -52,15 +52,16 @@ target_link_libraries(enclave "${SGX_COMMON_CFLAGS} -Wl,--no-undefined -nostdlib
 
 add_custom_command(TARGET enclave
     POST_BUILD
+    COMMENT "Create enclave signer's key"
+    COMMAND openssl genrsa -3 -out ${CMAKE_BINARY_DIR}/enclave_signer_private.pem 3072
```
Comment (mbrandenburger) [inline ecc_enclave/enclave/CMakeLists-common-app-enclave.txt]: why do we create a new signer key for every build?

Comment (g2flyer) [reply ecc_enclave/enclave/CMakeLists-common-app-enclave.txt]: > why do we create a new signer key for every build?

You forgot we are Intel, we want users to use as much cycles as possible  ... ;-)

**Comment by bvavala** [comment]
* There is no need to ship those keys, particularly private ones, with the repo - so, yes, it is a good idea to remove them. 
* We have no requirement that a user/signer has to have such key - so it's easier not to impose users to make their own.
  * Similarly, it is not the case to keep a key lying around when it's not necessary - creating it in the build folder forces an appropriate cleanup when artifacts are removed.
* We have no requirement that such key has to be the same for all enclaves created locally.
* Also, we have no (strong) performance requirements for enclave build times.

**Comment by mbrandenburger** [comment]
ok, please rebase and I will merge.

**Pull request merged**